snippets/current/package.json

```diff
@@ -8,27 +8,29 @@
     "dev:mediator-setup": "ts-node --esm src/mediator-setup.ts",
     "dev:mediation-recipient": "ts-node --esm src/mediation-recipient.ts",
     "dev:connection": "ts-node --esm src/create-a-connection.ts",
-    "dev:credential": "ts-node --esm src/issue-a-credential.ts",
+    "dev:issue-anoncreds-credential-didcomm": "ts-node --esm src/issue-anoncreds-credential-didcomm.ts",
+    "dev:openid4vc": "ts-node --esm src/sd-jwt-vc-openid4vc.ts",
     "dev:register-with-indy": "ts-node --esm src/register-schema-and-cred-def.ts",
     "dev:set-up": "ts-node --esm src/set-up.ts"
   },
   "dependencies": {
-    "@credo-ts/anoncreds": "0.5.0-alpha.110",
-    "@credo-ts/askar": "0.5.0-alpha.110",
-    "@credo-ts/core": "0.5.0-alpha.110",
-    "@credo-ts/indy-vdr": "0.5.0-alpha.110",
-    "@credo-ts/cheqd": "0.5.0-alpha.110",
-    "@credo-ts/node": "0.5.0-alpha.110",
-    "@credo-ts/react-native": "0.5.0-alpha.110",
-    "@hyperledger/anoncreds-nodejs": "0.2.0-dev.9",
-    "@hyperledger/anoncreds-react-native": "0.2.0-dev.9",
-    "@hyperledger/anoncreds-shared": "0.2.0-dev.9",
-    "@hyperledger/aries-askar-nodejs": "0.2.0-dev.6",
-    "@hyperledger/aries-askar-react-native": "0.2.0-dev.6",
-    "@hyperledger/aries-askar-shared": "0.2.0-dev.6",
-    "@hyperledger/indy-vdr-nodejs": "0.2.0-dev.6",
-    "@hyperledger/indy-vdr-react-native": "0.2.0-dev.6",
-    "@hyperledger/indy-vdr-shared": "0.2.0-dev.6",
+    "@credo-ts/anoncreds": "^0.5.3",
+    "@credo-ts/askar": "^0.5.3",
+    "@credo-ts/cheqd": "^0.5.3",
+    "@credo-ts/core": "^0.5.3",
+    "@credo-ts/indy-vdr": "^0.5.3",
+    "@credo-ts/node": "^0.5.3",
+    "@credo-ts/openid4vc": "^0.5.3",
```

Comment:
```suggestion
    "@credo-ts/openid4vc": "0.5.3",
```

Reply:
Would like to keep it with latest version for the scripts